services/galley/src/Galley/Data.hs

```diff
@@ -188,6 +190,16 @@ teamMembers t = mapM newTeamMember' =<<
     newTeamMember' (uid, perms, minvu, minvt, mlhStatus) =
         newTeamMemberRaw uid perms minvu minvt (fromMaybe UserLegalHoldDisabled mlhStatus)
 
+teamMembers' :: forall m. (MonadThrow m, MonadClient m) => TeamId -> [UserId] -> m [TeamMember]
```

Comment:
What's the difference between `teamMembers'` and `teamMembers`? Shouldn't one use the other? Maybe add a comment here.

Reply:
I renamed `teamMembers'` to `teamMembersLimited` with a comment, hope that's more clear. One cannot reuse the other since the CQL query needs to look quite different.